### ISSUE TITLE
I need a working example of #destroy_many

### ISSUE THREAD

**Issue by josephecombs** [zendesk/zendesk_api_client_rb#262]
We have tests which are iterating over n tickets and making n network requests.  I want to make one request using the #destroy_many method that apparently exists and can be passed an array of ticket IDs.  Can I please be shown a working code sample of this method?  Is this method called on an instance of Zendesk::Client?

**Comment by steved** [comment]
Looks like the proper modules weren't included on the Ticket class. I've released `1.11.4` to fix that.

This is a pattern we've somewhat implemented internally, but I haven't added helpers for JobStatus yet. You may also want to have a timeout wrapping the loop.

```ruby
job_status = client.tickets.destroy_many!([1, 2, 3])

while [nil, 'queued', 'working'].include?(job_status.status)
  sleep(0.25)
  job_status.reload!
end

job_status.status # => completed
```

**Comment by josephecombs** [comment]
Awesome.  Thanks @steved !!
